src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -81,6 +109,22 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
                 arbeidsuforhet_aktiv_vurdering = EXCLUDED.arbeidsuforhet_aktiv_vurdering,
                 sist_endret = EXCLUDED.sist_endret
             """
+
+        private const val UPSERT_PERSON_OVERSIKT_STATUS_MER_OPPFOLGING =
```

Comment:
Denne pratet vi kanskje om i sonen🤔 Synes vi bør være spesifikke hvilket domene det er snakk om 😊
```suggestion
        private const val UPSERT_PERSON_OVERSIKT_STATUS_SEN_OPPFOLGING =
```